content/zh/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -244,13 +250,15 @@ Pod 模板就像饼干切割器，而不是指定所有副本的当前期望状
 
 {{% /capture %}}
 
```

Comment:
原文 234 行中的 
```
Rather than specifying the current desired state of all replicas, pod templates are like cookie cutters. Once a cookie has been cut, the cookie has no relationship to the cutter. 
```
这里的翻译可以优化吗